examples/AdminClient/Program.cs

```diff
@@ -84,7 +85,202 @@ static async Task CreateTopicAsync(string bootstrapServers, string topicName)
                 }
                 catch (CreateTopicsException e)
                 {
-                    Console.WriteLine($"An error occured creating topic {e.Results[0].Topic}: {e.Results[0].Error.Reason}");
+                    Console.WriteLine($"An error occurred creating topic {e.Results[0].Topic}: {e.Results[0].Error.Reason}");
+                }
+            }
+        }
+
+        static List<T> ParseAclBindings<T>(string[] args) where T: AclBinding, new()
```

Comment:
although the inheritance relationship may be convenient, this comes across as a bit weird to me. see comment elsewhere. i notice the Java implementation has a toFilter method on the AclBinding class, so probably do similar instead and make use of that i think.